packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -66,8 +73,13 @@ const NodeProtoLastChildGetter: Getter = ReflectApply(ObjectProtoLookupGetter, N
 const blueDocumentToBlueCreateHooksCallbackMap = toSafeWeakMap(
     new WeakMapCtor<Document, Connector>()
 );
+// @ts-ignore: Prevent cannot find name 'ShadowRealm' error.
+const ShadowRealmCtor = SUPPORTS_SHADOW_REALM ? ShadowRealm : undefined;
+const ShadowRealmProtoEvaluate: CallableEvaluate | undefined = ShadowRealmCtor?.prototype?.evaluate;
```

Comment:
So there seem to be some new government regulation in Europe countries to prohibit unsafe-eval in CSP (https://salesforce-internal.slack.com/archives/C014RRJ9DKP/p1661973460351309). This could break us even if we move to ShadowRealm if we use `evaluate()`

Reply:
We will need to look at Aura loader as well, to see where the fiber breaks... it is very likely that in the loader, we are operating using a promise, which give us the possibility to use importValue, or something like that. This is interesting.